src/javascripts/cookies.js

```diff
@@ -0,0 +1,40 @@
+// Add cookie to session
```

Comment:
Would it be worth having cookie-helper.js (with the get, set cookie functions) and a separate one for actually doing cookie-based things, such as hiding the cookie notice?

Reply:
Yep definitely worth doing that, but will refactor together with other helpers.